src/EA.Weee.Web/Areas/Producer/Views/Producer/CheckAnswers.cshtml

```diff
@@ -7,9 +15,169 @@
+        if (representingOrg)
+        {
+            representingCompanyDetails = new Dictionary<string, object>
+{
+                { "Producer name", Model.RepresentingCompanyDetails.CompanyName },
+                { "Trading name", Model.RepresentingCompanyDetails.BusinessTradingName },
+                { "Address line 1", Model.RepresentingCompanyDetails.Address.Address1 },
+                { "Address line 2", Model.RepresentingCompanyDetails.Address.Address2 },
+                { "Town or city", Model.RepresentingCompanyDetails.Address.TownOrCity },
+                { "County or region", Model.RepresentingCompanyDetails.Address.CountyOrRegion },
+                { "Postcode", Model.RepresentingCompanyDetails.Address.Postcode },
+                { "Country", Model.RepresentingCompanyDetails.Address.CountryName },
+                { "Phone number", Model.RepresentingCompanyDetails.Address.Telephone },
+                { "Email", Model.RepresentingCompanyDetails.Address.Email },
+            };
+        }
+    }
+
+    <div>
+        @this.WeeeGds().TableSummary("Organisation details", organisationDetails, "", "", displayCaption: true)
+        <div style="margin-bottom: 40px;">
+            <a class="govuk-link govuk-link--no-visited-state" href="@Url.Action("EditOrganisationDetails", "ProducerSubmission", new { redirectToCheckAnswers = true})" style="font-size: 19px;">Change organisation details</a>
+        </div>
+    </div>
+
+    <div>
+        @this.WeeeGds().TableSummary("Contact details", contactDetails, "", "", displayCaption: true)
+        <div style="margin-bottom: 40px;">
+            <a class="govuk-link govuk-link--no-visited-state" href="@Url.Action("EditContactDetails", "ProducerSubmission", new { redirectToCheckAnswers = true})" style="font-size: 19px;">Change contact details</a>
+        </div>
+    </div>
+
+    <div>
+        @this.WeeeGds().TableSummary("Service of notice details", serviceOfNoticeData, "", "", displayCaption: true)
+        <div style="margin-bottom: 40px;">
+            <a class="govuk-link govuk-link--no-visited-state" href="@Url.Action("ServiceOfNotice", "ProducerSubmission", new { redirectToCheckAnswers = true})" style="font-size: 19px;">Change service of notice details</a>
+        </div>
+    </div>
+
+    @if (representingOrg)
+    {
+        <div>
+            @(this.WeeeGds().TableSummary("Represented company details", representingCompanyDetails, "", "", displayCaption: true))
+            <div style="margin-bottom: 40px;">
+                <a class="govuk-link govuk-link--no-visited-state" href="@Url.Action("EditRepresentedOrganisationDetails", "ProducerSubmission", new { redirectToCheckAnswers = true})" style="font-size: 19px;">Change represented company details</a>
+            </div>
+        </div>
+    }
+
+    <table class="govuk-table category-row-no-break">
+        <caption class="govuk-table__caption" style="font-size: 24px">Eee details</caption>
```

Comment:
@will-hope98  There are some font size classes https://design-system.service.gov.uk/styles/font-override-classes/ that can be used in the file rather than setting fixed size.

Reply:
Done